LedRGB.cpp

```diff
@@ -9,21 +9,15 @@ LedRGB::LedRGB(unsigned redPin, unsigned greenPin, unsigned bluePin)
 }
 
 void LedRGB::setRGB(unsigned r, unsigned g, unsigned b) {
-    r /= 2.55;
-    g /= 2.55;
-    b /= 2.55;
-    if (checkBrightness) {
-        RedLed.setBrightness(r);
-        GreenLed.setBrightness(g);
-        BlueLed.setBrightness(b);
-    }
+    setBrightness(r);
+    setBrightness(g);
+    setBrightness(b);
```

Comment:
Is the overall brightness an average of the 3 individual brightness? Or does the overall brightness effect the individual brightness in any way?